### ISSUE TITLE
Core: Choosing the starting area "Isle of Exile" doesn't work.

### ISSUE THREAD

**Issue by osleyder85** [TrinityCore/TrinityCore#25711]
**Description:**
When creating a new player, you must choose in which zone you want to start. When the "Isle of Exile" zone is chosen, send the player to the other zone. I think that to have a better gaming experience this should work, even if it is not developed. It is even necessary to send us to the "Isle of Exile" to fix it

**Steps to reproduce the problem:**

1. Create any player  
2. Choose the starting area "Isle of Exile"

**Branch(es):** master

**TC rev. hash/commit:** 
 364f6ee

**Operating system:**
Edition: Windows 10 Pro version 20H2, Version OS: 19042.662

**Comment by MaxtorCoder** [comment]
Yeah it seems Shauren hasn't implemented the spawn points for Horde/Alliance yet and isn't utilizing the bit in `CMSG_CREATE_CHAR` yet.

**Comment by mdX7** [comment]
added in https://github.com/TrinityCore/TrinityCore/commit/d54edfeb8bd9e525bac62677b8f9dffbaa5124c3
